lib/rack/test/cookie_jar.rb

```diff
@@ -12,7 +12,7 @@ class Cookie # :nodoc:
 
       # The name of the cookie, will be a string
       attr_reader :name
-      
+
```

Comment:
Did your editor change the whitespace? I'd be tempted to exclude these changes so they don't distract from the matter at hand.